marlowe-runtime-web/src/Language/Marlowe/Runtime/Web/Types.hs

```diff
@@ -334,14 +336,36 @@ instance ToJSON BlockHeader
 instance FromJSON BlockHeader
 instance ToSchema BlockHeader
 
-data CreateTxBody = CreateTxBody
+data CreateTxEnvelope tx = CreateTxEnvelope
```

Comment:
@jhbertra did I over complicate this type (`CreateTxEnvelope tx`) and I should split this into two types: `CreateTxBody` and `CreateTx` and duplicate related types as well or can I keep it as it is?